src/css/skins/five.less

```diff
@@ -133,4 +131,24 @@
         border-radius: @five-border-radius;
     }
 
+    // override control bar element colors that don't look good in time slider above mode
+    &.jw-flag-time-slider-above,
+    &.jw-breakpoint-0,
+    &.jw-breakpoint-1 {
+      .jw-controlbar {
+        .jw-text {
```

Comment:
Couldn't we just set the `@inactive-color` and `@hover-color` vars for `.jw-flag-time-slider-above` etc for both skins?